learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -75,31 +75,31 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Typo tolerant  | ✅ | ✅ | ✅ | 🔶 <br>Needs to be specified by fuzzy queries |
-| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Tie-breaking order is limited by a unique scoring rule | ❌|
-| Custom rules | ✅ | ✅ | 🔶 <br>Limited to one default sorting rule | 🔶 <br>Function score query
+| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Field priority can be changed, but ranking rules order cannot be changed. | ❌|
```

Comment:
I initially didn't understand what you meant by the earlier statement, but I read up on this and I get it now :)

So I've updated the description to be more precise.

Reply:
I cannot review this, we don't use `Field priority`.
@gmourier is this a common idiom in the search engine industry? Otherwise what would you use so that readers who are more familiar with Meilisearch will understand what is written here?

Reply:
Looks like the equivalent terminology in Meilisearch is [`Attribute Ranking Order`](https://docs.meilisearch.com/learn/core_concepts/relevancy.html#attribute-ranking-order).

Reply:
`Field priority` seems ok to me; maybe `Field weight` could be clearer?

Reply:
`Field weight` seems like the best option, since it returns the most relevant results on Google (and that's where confused readers tend to go, in my experience).

Reply:
Thanks for your input @gmourier @dichotommy 
@jasonbosco would you mind using `Field weight`? thx

Reply:
Done.